src/test/app/common/country.ts

```diff
@@ -0,0 +1,39 @@
+/* tslint:disable:no-unused-expression */
+
+import * as chai from 'chai'
+import * as chaiAsPromised from 'chai-as-promised'
+chai.use(chaiAsPromised)
+
+import { Country } from 'app/common/country'
+
+describe('Country', () => {
+  describe('valueOf', () => {
+    it('should return undefined for unknown country', () => {
+      chai.expect(Country.valueOf('unknown-country')).to.be.undefined
+    })
+
+    it('should return country for known country', () => {
+      Country.all().forEach(country => {
+        chai.expect(Country.valueOf(country.value)).to.be.equal(country)
+      })
+    })
+  })
+
+  describe('isClaimantCountry', () => {
```

Comment:
Please add missing tests cases for other methods you implemented in the `country.ts` file.